public/js/main.js

```diff
@@ -185,9 +185,34 @@ filters.addEventListener("click",(event) => {
   const filterSelected = document.getElementsByClassName("selected")[0];
   filterSelected.removeAttribute("class");
   element.className = "selected";
-  renderData();
+  todoListComputed();
 });
 
+listData.addEventListener("dblclick",(event) => {
+  const element = event.target;
+  const item = element.parentNode.parentNode;
+  item.classList.add("editing");
+  const inputEdit = item.lastElementChild;
+
+  inputEdit.onkeypress = (e) => {
+    const enterKey = 13;
+    if (e.charCode === enterKey && inputEdit.value.trim() !== "") {
+      const taskId = item.getAttribute("id");
+      todoList.map(item => {
```

Comment:
`map` will return an array data, however, you didn't assign to any variable, just use it as a loop, should replace to `foreach`